gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/processor/notfound/NotFoundReporter.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.reactor.processor.notfound;
+
+import io.gravitee.common.http.HttpHeaders;
+import io.gravitee.gateway.api.ExecutionContext;
+import io.gravitee.gateway.core.processor.AbstractProcessor;
+import io.gravitee.gateway.report.ReporterService;
+import io.gravitee.reporter.api.log.NotFound;
+
+public class NotFoundReporter extends AbstractProcessor<ExecutionContext> {
+
+    private final ReporterService reporterService;
+
+    public NotFoundReporter(final ReporterService reporterService) {
+        this.reporterService = reporterService;
+    }
+
+    @Override
+    public void handle(ExecutionContext context) {
+        NotFound reportMessage = new NotFound(System.currentTimeMillis());
```

Comment:
I don't understand the purpose of this.

Why would we need to create a dedicated Reportable for unmapped request ?
I would prefer to make use of the `Metrics` object for which we are setting a dummy api-id (as we are doing for an unknown application (id = 1)).

WDYT ?

Reply:
Well, I thought it would be confusing to use the same object - there are no proxy requests/response (return 0-times?) and other fields (no proxies etc). Do you think it's not a good approach?

Reply:
I would prefer to use existing Metrics just to avoid too many reporter types.
And, I would assume, that if the request is mapped to an API or not... that's still a request, so no reason to change this, IMHO.

Reply:
ok, I'll fix this tomorrow